apps/sms/views/shared/test/unit/activity_handler_test.js

```diff
@@ -345,22 +345,20 @@ suite('ActivityHandler', function() {
       var sendSpy;
       var findByPromise;
 
-      setup(function() {
+      setup(function(done) {
         sendSpy = this.sinon.spy(window, 'Notification');
         findByPromise = Promise.resolve(
           [ { name: [contactName] } ]
         );
         this.sinon.stub(Contacts, 'findByAddress').returns(findByPromise);
 
-        MockNavigatormozApps.mTriggerLastRequestSuccess();
+        MockNavigatormozApps.mTriggerLastRequestSuccess().then(done, done);
```

Comment:
I think it should be more correct way to wait for the overall action finished if we return a promise in success request. wdyt?